app/views/layouts/design_system.html.erb

```diff
@@ -0,0 +1,54 @@
+<% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
```

Comment:
Rather than adding a new layout we should use the component for admin layout: https://components.publishing.service.gov.uk/component-guide/layout_for_admin